crates/client/src/test_generator.rs

```diff
@@ -123,7 +123,14 @@ impl TestGenerator {
             env!("CARGO_MANIFEST_DIR"),
             "/src/templates/trdelnik-tests/test.rs"
         ));
-        self.create_file(&test_path, TESTS_FILE_NAME, test_content)
+        let program_libs = self.get_program_lib_names(root).await?;
+        let program_name = if let Some(name) = program_libs.first() {
+            name
+        } else {
+            throw!(Error::CannotParseCargoToml)
```

Comment:
My only minor concern is that we throw an error here, but later, while generating fuzz test files, we continue within the same "else" branch without throwing an error. I understand that if obtaining names fails, it will throw an error here and won't even get to the fuzz test generation. However, I would suggest updating it for consistency.

Reply:
Fair point, fixed.